### PR TITLE
Fix usage example in phrases.py

### PR DESCRIPTION
**Pull request by piskvorky**
Fixing poor English, clarifying exposition.

### CODE REVIEW

**Comment by piskvorky** [comment]
@mpenkov it's not clear to me why the tests failed. Should I fix my changes somehow?

**Comment by mpenkov** [comment]
@piskvorky Sorry for the late response.

Check out the builds here: https://travis-ci.org/RaRe-Technologies/gensim/builds/570161971?utm_source=github_status&utm_medium=notification - flake8 is not happy about the line length.

> gensim/models/phrases.py:6:121: E501 line too long (125 > 120 characters)

File: gensim/models/phrases.py [hunk 1]
```diff
@@ -20,19 +23,38 @@
...
-    [u'trees_graph', u'minors']
+    >>> # The training corpus must be a sequence (stream, generator) of sentences,
+    >>> # with each sentence a list of tokens:
+    >>> print(list(sentences)[0][:10])
+    ['computer', 'human', 'interface', 'computer', 'response', 'survey', 'system', 'time', 'user', 'interface']
+    >>>
+    >>> # Train a toy bigram model.
+    >>> phrases = Phrases(sentences, min_count=1, threshold=1)
+    >>> # Apply the trained phrases model to a new, unseen sentence.
+    >>> phrases[['trees', 'graph', 'minors']]
```
Comment (mpenkov) [inline gensim/models/phrases.py]:
```suggestion
    >>> phrases[['trees', 'graph', 'minors']]
```

Comment (piskvorky) [reply gensim/models/phrases.py]: Ah, editing directly in Github editor. It doesn't remove trailing whitespace apparently. Nice catch.

File: gensim/models/phrases.py [hunk 2]
```diff
@@ -20,19 +23,38 @@
...
+    ['trees_graph', 'minors']
+    >>> # The toy model considered "trees graph" a single phrase => joined the two
+    >>> # tokens into a single token, `trees_graph`.
     >>>
-    >>> phrases.add_vocab([["hello", "world"], ["meow"]])  # update model with new sentences
+    >>> # Update the model with two new sentences on the fly.
+    >>> phrases.add_vocab([["hello", "world"], ["meow"]])
```
Comment (mpenkov) [inline gensim/models/phrases.py]:
```suggestion
    >>> phrases.add_vocab([["hello", "world"], ["meow"]])
```

**Pull request merged**